pkgs/applications/audio/picard/default.nix

```diff
@@ -1,16 +1,18 @@
-{ stdenv, python3Packages, fetchFromGitHub, gettext, chromaprint, qt5
+{ stdenv, fetchFromGitHub, gettext, chromaprint, qt5 # general deps
+, buildPythonApplication, mutagen, discid, dateutil  # python deps
+, pyqt5 ? null, pyqt5_with_qtmultimedia ? null       # optional python deps
 , enablePlayback ? true
 , gst_all_1
-}:
+}@args:
```

Comment:
This (`@args`) is debatable. The alternative would be using `python3Packages`.

Reply:
Use `python3Packages` in the inputs, otherwise I'm not sure `.override` can work. <sup>\[citation needed\]</sup>

Reply:
Ok. I'm using pythonPackages now. This refers always to the current python in a python derivation (called with `python*.callPackages`, while python3Packages is inherited from the global scope.